src/main/java/com/coveros/selenified/application/Check.java

```diff
@@ -752,22 +744,22 @@ default String checkCookieEquals(String cookieName, String expectedCookieValue,
      * @param timeTook              - the amount of time it took for wait for something (assuming we had to wait)
      * @return String: if the cookie is present, the value of the cookie, and null if the cookie isn't present
      */
-    default String checkCookieMatches(String cookieName, String expectedCookiePattern, double waitFor, double timeTook) {
+    String checkCookieMatches(String cookieName, String expectedCookiePattern, double waitFor, double timeTook) {
         // check for the object to be present
         String cookieValue;
-        boolean isCookiePresent = getApp().is().cookiePresent(cookieName);
+        boolean isCookiePresent = this.app.is().cookiePresent(cookieName);
         if (isCookiePresent) {
-            cookieValue = getApp().get().cookieValue(cookieName);
+            cookieValue = this.app.get().cookieValue(cookieName);
         } else {
-            getReporter().fail("to find cookie with the name <b>" + cookieName + "</b> and a value matching pattern of <b>" + expectedCookiePattern + STORED,
+            this.reporter.fail("to find cookie with the name <b>" + cookieName + "</b> and a value matching pattern of <b>" + expectedCookiePattern + STORED,
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "</b> and a value matching pattern of <b>" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)